plugins/CMakeLists.txt

```diff
@@ -55,6 +55,17 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
 	set(EXTRA_WIN_LIBRARIES winpthread)
 endif()
 
+if("${LIBAD9166_LIBRARIES}" STREQUAL "")
+	message(WARNING "libad9166 not found. The following plugins will not be installed: cn0511")
+	list(REMOVE_ITEM PLUGINS cn0511)
+endif()
+
+if("${LIBAD9361_LIBRARIES}" STREQUAL "")
+	message(WARNING "libad9361 not found. The following plugins will not be installed: fmcomms2
+		fmcomms2_adv spectrum_analyzer")
```

Comment:
nit: maybe put the plugins dependent on this libraries in a variable so that it's easier to handle if we ever need to extend the list or use that list anywhere else

Reply:
I agree.